gradle/libs.versions.toml

```diff
@@ -217,6 +218,7 @@ kotlinter-gradlePlugin = { module = "org.jmailen.gradle:kotlinter-gradle", versi
 
 # lint dependencies
 lintApi = { module = "com.android.tools.lint:lint-api", version.ref = "lintApi" }
+androidx-core = { group = "androidx.core", name = "core", version.ref = "core" }
```

Comment:
it's new?

Reply:
It does look suspect, I am not sure if I added that myself or if the IDE did when I accidentally pressed Option Enter to auto-resolve some import. And it instead brought in some dependency.
I will take a look at what happens if I remove it